### DISCUSSION TITLE
Difference to @react-google-maps/api

### DISCUSSION THREAD

**Discussion by Fabioni** [visgl/react-google-maps#163] [General]
Hey,
I wonder what the difference is between this package and https://www.npmjs.com/package/@react-google-maps/api, that resonates the redevelopment instead of combining workforce :)

**Comment by apeTrackunit** [comment]
You might find the answer here: [Introducing React Components for the Maps JavaScript API](https://cloud.google.com/blog/products/maps-platform/introducing-react-components-for-the-maps-javascript-api). It appears that Google has decided to prioritize Google Maps integration and collaborate with visgl to provide support for React developers.

**Comment by usefulthink** [comment]
There's a lot of different aspects to this. As a quick summary:

From an organisational point of view, I think one of the more important parts was that we didn't want to put our time and effort into a project that is controlled and privately owned by another individual (which is the case for all previously existing google-maps/react integrations I am aware of). This is why after a lot of discussions we ended up with making this project part of the vis.gl organisation in the OpenJS foundation.

Architecturally, while similar to other implementations, ours tries to stay close to [react-map-gl](https://www.npmjs.com/package/react-map-gl) and aims to provide a fully reactive wrapper for the maps API (parts of this still in development) that integrates well with deck.gl for example.
